### PR TITLE
Kotlin lsp language configuration

### PR DESCRIPTION
**Pull request by joncarr**

### CODE REVIEW

File: lsp-clients.el [hunk 1]
```diff
@@ -659,5 +659,27 @@ finding the executable with `exec-path'."
...
 

+
+;; Kotlin
+(defgroup lsp-kotlin nil
+  "Kotlin."
+  :group 'lsp-mode
+  :tag "Kotlin")
+
+(defcustom lsp-clients-kotlin-server-command
```
Comment (yyoncho) [inline lsp-clients.el]: this one is not used(you could remove the group as well).

**Comment by yyoncho** [comment]
Can you resolve the conflicts with README.org?

File: lsp-clients.el (hunk 1 above)
Comment (joncarr) [reply lsp-clients.el]: removed `lsp-clients-kotlin-server-commad`

**Comment by joncarr** [comment]
resolved readme coflict (my bad :))

**Comment by yyoncho** [comment]
Thank you! :heart: 
And congrats for your first `lsp-mode` PR! :tada:

**Pull request merged**